com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,38 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: rack2free
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 metainfo.xml ${FLATPAK_DEST}/share/${FLATPAK_ID}.metainfo.xml
+      - install -Dm755 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644  res/icon.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/com.vcvrack.Rack2.png
+      - install -Dm755 rack_wrapper.sh ${FLATPAK_DEST}/bin/rack
+      - cp -a ./ ${FLATPAK_DEST}/
+    sources:
+      - type: file
+        path: com.vcvrack.Rack2.desktop
+      - type: file
+        path: metainfo.xml
+      - type: script
+        dest-filename: rack_wrapper.sh
+        commands:
+          - cd /app
+          - exec /app/Rack --system /app --user ${XDG_DATA_HOME} $@
```

Comment:
```suggestion
          - exec /app/Rack --system /app --user ${XDG_DATA_HOME} "$@"
```

Reply:
also the exectuable `/app/Rack` is in the wrong location

Reply:
> also the exectuable `/app/Rack` is in the wrong location

Executable is exactly where it intended by developers to be: https://vcvrack.com/manual/Installing#Installing-on-Linux. To call it I'm using wrapper script.